ds-card-reader/driver/driver_test.go

```diff
@@ -307,72 +307,3 @@ func TestHandleWriteCommands(t *testing.T) {
 		})
 	}
 }
-
-func TestCardReaderDriver_Initialize(t *testing.T) {
```

Comment:
Why removing this test? It must stay and be update appropriately. See changes made to ds-controller-board tests